package.json

```diff
@@ -10,25 +10,32 @@
     "lint": "tslint -p ."
   },
   "dependencies": {
+    "@aerogear/app": "dev",
+    "@aerogear/auth": "dev",
+    "@aerogear/push": "dev",
+    "@aerogear/security": "dev",
+    "@aerogear/voyager-client": "dev",
     "@aerogear/voyager-server": "dev",
     "@types/chai": "4.1.7",
     "@types/express": "4.17.0",
     "@types/mocha": "5.2.7",
     "@types/node": "12.6.9",
-    "axios": "^0.19.0",
-    "chai": "^4.2.0",
-    "express": "latest",
-    "graphql": "latest",
-    "mocha": "^6.1.4",
-    "node-fetch": "^2.6.0",
-    "pg": "^7.11.0",
+    "@types/node-fetch": "2.5.0",
+    "@types/pg": "7.4.14",
+    "axios": "0.19.0",
+    "chai": "4.2.0",
+    "express": "4.17.1",
+    "graphql": "14.4.2",
```

Comment:
Should not we use latest for `express` and `graphql`? That is what user will eventually install.

Reply:
Yes, ist's better to test with the latest version of graphql and express, but at the same time if we always use the latest it will be difficult to understand if a new issue will be caused by a new version of express/graphql or to new changes to the aerogear packages, in any case, this problem could need a wider analysis so for now I will revert it to `latest`